observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+Dashboard provides "at glance" view of Key Performance Index(KPI) in accrodance with goals and objectives of business processes. Typically this includes metrices and key data points to monitor the health of a business, department or specific process. Dashboard usually connects to one or multiple data sources allowing the creation of visual representation of data. This is important in observability as a good dashboard will be able to 
```

Comment:
1) Nit: `A` dashboard provides... or Dashboard`s` provide.
2) Nit: metrics instead of metric`e`s
3) Nit: `accordance`